File: app/src/main/java/kryx07/expensereconcilerclient/db/MyDatabase.java
package kryx07.expensereconcilerclient.db;

/**
 * Created by sda on 05.07.17.
 */


import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import kryx07.expensereconcilerclient.model.persontest.Person;

@Database(entities = {Person.class}, version = 1)
public abstract class MyDatabase extends RoomDatabase {
    abstract PersonDao personDao();
}

File: app/src/main/java/kryx07/expensereconcilerclient/db/MyDatabase.kt
package kryx07.expensereconcilerclient.db

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import android.arch.persistence.room.TypeConverters
import kryx07.expensereconcilerclient.model.transactions.Payable
import kryx07.expensereconcilerclient.model.transactions.Payables
import kryx07.expensereconcilerclient.model.transactions.Transaction

@Database(entities = arrayOf(Transaction::class, Payable::class), version = 5)
@TypeConverters(*arrayOf(Converters::class))
abstract class MyDatabase : RoomDatabase() {

    abstract fun transactionDao(): TransactionDao

    abstract fun payablesDao(): PayablesDao
}

